Robot-Tests/testcases_Rautatien_huoltotie.robot

```diff
@@ -0,0 +1,104 @@
+#    Joni Laari    Sitowise    2024
+
+*** Settings ***
+Documentation       Regression testcases for Digiroad
+Resource            common_keywords.robot
+
+Suite Setup         Login To DigiRoad
+Suite Teardown      Close Browser
+
+Test Setup          Testin Aloitus
+
+*** Variables ***
+@{rautatien_huoltotiet_radionappi}    responsibility    rightOfUse
+@{Huoltotie_käyttöoikeus_kohteet}    Tieoikeus    Tiekunnan osakkuus    LiVin hallinnoimalla maa-alueella    Kävelyn ja pyöräilyn väylä    Muu sopimus    Potentiaalinen käyttöoikeus    Tuntematon
+@{Huoltotie_käyttöoikeus_numerot}    1    2    3    4    6    9    99
+@{Huoltotie_huoltovastuu_numerot}    1    2    99
+
+
+#responsibility/rightofuse korjattava takaisin kun testit toimii 19.9
+*** Test Cases ***
+Rautateiden huoltotien tarkastelu
```

Comment:
Yleinen kommentti näistä for loopeista että, jos loopattaisiin tuolla keywordin puolella, niin menisi varmaan nopeammin? En nyt pelkästään koodia katsomalla oikein hahmota, tuoko tuo testin aloitus, zoomailu yms. jotain lisäarvoa itse testiin.

Reply:
Perusteltu korjaus, mutta sen voisi backlogittaa ja viedä myöhemmin suoraan qa:lle (myös muiden tietolajien testeille jos sieltä löytyy vastaavia looppeja). 

Kokeilin korjata heti tähän mergeen mutta ei lähtenyt suoriltaan toimimaan.

Reply:
Sen verran vielä, että Testin Aloitus, zoomailu etc. ovat keinoja välttää linkkien ohiklikkaukset, epäonnistuneet takedownit yms testityökalun tuottamat tarpeettomat failit.